src/cnfizers/Cnfizer.h

```diff
@@ -62,7 +62,6 @@ class Cnfizer
     virtual ~Cnfizer( ) { }
 
     lbool cnfizeAndGiveToSolver (PTRef, FrameId frame_id); // Main routine
-    void  declareToSolver(const vec<PTRef>& nestedBoolRoots, FrameId frame_id); // Entry point for declaring.
     void  declareVars    (vec<PTRef>&); // Declare a set of Boolean atoms to the solver (without asserting them)
```

Comment:
Can we make this private?

Reply:
If you mean declareVars than yes. Should it be private or protected?

Reply:
Made protected in 4cded77